### PR TITLE
Refactor tests to reduce snapshot

### LINKED ISSUE #552: Reduce size of InterventionPlan/IRS snapshot test
This snapshot test is too large: `src/containers/pages/InterventionPlan/IRS/tests/__snapshots__/IRS.test.tsx.snap`

### PR DESCRIPTION
**Pull request by ciremusyoka**
Closes #552

### CODE REVIEW

**Comment by ciremusyoka** [comment]
@moshthepitt  Is it necessary to have have snapshots if the test coverage is `100%` even without them?

**Comment by moshthepitt** [comment]
@ciremusyoka in some cases yes because Snapshot tests serve a different purpose.  Please see: https://kentcdodds.com/blog/effective-snapshot-testing/

File: src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx [hunk 1]
```diff
@@ -64,7 +65,6 @@ describe('containers/pages/IRS', () => {
     );
     const helmet = Helmet.peek();
     expect(helmet.title).toEqual(IRS_PLANS);
-    expect(toJson(wrapper)).toMatchSnapshot();
```
Comment (cKellyDesign) [inline src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx]: Agreed with @moshthepitt, let's add this line back but find some child of wrapper which is most relevant to the tests (so we're not saving a snapshot of the whole app).

**Comment by moshthepitt** [comment]
@ciremusyoka please rebase/update so that we can merge this.

File: src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx [hunk 2]
```diff
@@ -64,7 +63,35 @@ describe('containers/pages/IRS', () => {
...
+    expect(wrapper.find('IrsPlans').props()).toMatchSnapshot('irs plans props');
+    expect(wrapper.find('DrillDownTable').props()).toMatchSnapshot('drill down table props');
+    expect(wrapper.find('Button').text()).toEqual('Create New Plan');
+    expect(wrapper.find('Button').props()).toMatchSnapshot('button props');
```
Comment (cKellyDesign) [inline src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx]: 🎉

File: src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx [hunk 3]
```diff
@@ -64,7 +63,35 @@ describe('containers/pages/IRS', () => {
...
+    wrapper.unmount();
+  });
+
+  it('render loader when plansArray is empty', () => {
```
Comment (cKellyDesign) [inline src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx]: ❤️

**Pull request merged**